assets/css/activitypub-admin.css

```
@@ -230,13 +230,15 @@ input.blog-user-identifier {
 	font-family: dashicons;
 }
 
-.repost .dashboard-comment-wrap,
-.like .dashboard-comment-wrap {
+.like .dashboard-comment-wrap,
+.quote .dashboard-comment-wrap,
+.repost .dashboard-comment-wrap {
```

```suggestion
.activitypub-comment .dashboard-comment-wrap {
```
We could also do that and be more forward compatible.